src/udiskssimplejob.c

```diff
@@ -109,8 +109,10 @@ udisks_simple_job_complete (UDisksSimpleJob     *job,
                             gboolean             success,
                             const gchar         *message)
 {
+  static const gchar none[] = "";
+
   g_return_if_fail (UDISKS_IS_SIMPLE_JOB (job));
-  udisks_job_emit_completed (UDISKS_JOB (job), success, message);
+  udisks_job_emit_completed (UDISKS_JOB (job), success, (message) ? message : none);
```

Comment:
Isn't a string constant/literal just a static constant anyway? That's I think it should make no difference. Thanks for changing it!